csrankings.csv

```diff
@@ -11356,7 +11360,7 @@ Malak Abdullah,Jordan University of Science and Technology,http://www.just.edu.j
 Malcolm D. Atkinson,University of Edinburgh,http://www.research.ed.ac.uk/portal/en/persons/malcolm-atkinson(8e51eafb-dd78-468b-8970-ec2349e92305).html,NOSCHOLARPAGE
 Malcolm Heywood,Dalhousie University,https://www.dal.ca/faculty/computerscience/faculty-staff/malcolm-heywood.html,_d-AGjUAAAAJ
 Malcolm P. Atkinson,University of Edinburgh,https://www.inf.ed.ac.uk/people/staff/Malcolm_Atkinson.html,NOSCHOLARPAGE
-Malcolm R. K. Ryan,Macquarie University,https://researchers.mq.edu.au/en/persons/malcolm-ryan,zETJIuoAAAAJ
+Malcolm R Ryan,Macquarie University,https://researchers.mq.edu.au/en/persons/malcolm-ryan,zETJIuoAAAAJ

```

Comment:
This is not a DBLP name. https://dblp.uni-trier.de/pers/hd/r/Ryan:Malcolm_R=_K= is. (It's easy to spot when you have added a name that is not in DBLP, because middle initials are always followed by a period.) You need to check every name against DBLP.